src/pykeen/evaluation/__init__.py

```diff
@@ -1,6 +1,71 @@
 # -*- coding: utf-8 -*-
 
-"""Evaluation."""
+"""Evaluation.
+
+Knowledge graph embeddings models are usually evaluated based on link prediction, which is defined
+as predicting the tail/head entities for $(h,r)$/$(r,t)$ pairs.
+
+However, given the fact that both the training and the test set usually only contain true facts, negative samples are generated by corrupting these positive triples.
+For each test triple $t^{+} = (h,r,t) \in \mathcal{K}_{test}$ two sets of corrupted triples are constructed:
+
+    * $\mathcal{H}(r, t) = \{(h', r, t) \mid h' \in \mathcal{E} - \{h\}$
+    * $\mathcal{T}(h, r) = \{(h, r, t') \mid t' \in \mathcal{E} - \{t\}\}$
+
+where $\mathcal{H}(r, t)$ contains all the triples where the head entity has been corrupted, and $\mathcal{T}(h, r)$
+contains all the triples with corrupted tail entities.
+
+Among the corrupted triples in $\mathcal{H}(r, t)$ / $\mathcal{T}(h, r)$, there might be true triples that are part of
```

Comment:
@mali-git you should read the stuff that I wrote in https://github.com/pykeen/pykeen/pull/401 and try and deduplicate the first part of this

Reply:
actually it's probably better to delete the first section up until the note, since it's duplicating stuff that was already elsewhere in the docs

Reply:
Please feel free to adapt/remove corresponding parts.

Reply:
Seems to be exceeded by a13aa17 and 29c89e4.